src/get-authorization-url.ts

```diff
@@ -21,6 +22,7 @@ async function getAuthorizationUrl(options: GetAuthURLOptions = {}) {
     screenHint,
     organizationId,
     loginHint,
+    prompt,
```

Comment:
This is failing because this library is on an older version of `@workos-inc/node`.

```json
"@workos-inc/node": "^7.37.1",
```